pkg/controller/storagecluster/storagecluster.go

```diff
@@ -465,6 +497,9 @@ func (c *Controller) runPreflightCheck(cluster *corev1.StorageCluster) error {
 			return err
 		}
 
+		// XXX - Sleep to avoid race cond check NOTE: in setStorageClusterDefaults()
+		time.Sleep(30 * time.Second)
+
```

Comment:
This is actually something that Jiafeng had me add for my testing to avoid the race condition.  At the time I was having issues getting the daemonset to start.    We can remove that for sure.   He mentioned that the same issue can happen here that is happening in setStorageClusterDefaults().